src/client/componentsMealsharing/Mealspecific.js

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from "react";
+import "./Meal.css";
+import ReservationSpecific from "./ReservationSpecific";
+
+let id = 0;
+let max_reservations = 0;
+function Mealspecific({ match }) {
+  const [fetchItem, setFetchItem] = useState([]);
+  const [id, setId] = useState(match.params.id);
```

Comment:
you don't really this state, you could just use the match.params.id directly where you call the api :)

Reply:
changed to  const id = match.params.id; :)